app/services/feeds/hacker_news/sourceable.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Feeds
+  module HackerNews
+    # Indicates the human-readable name for the feed source.
+    # TODO: We ultimately want to pull this from a FeedSource table.
+    module Sourceable
+      SOURCE_ID = 1
+      SOURCE = 'Hacker News news feed'
+      SOURCE_H = {
+        source_id: SOURCE_ID,
+        source: SOURCE,
+      }
+    end
+  end
+end
```

Comment:
A simple module that provides information about a news feed source, Hacker News in this case. In the real world, this would be stored in the database.